pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+addopts = "--doctest-glob=\"*.rst\""
```

Comment:
`--doctest-glob` activates pytest's doctest.

Back in #202 I caught a lot of doctest mismatch with actual behavior, so activating this is good :-)